samples/charts/data-chart/category-series/Pages/DataChartTypeCategorySeries.razor

```diff
@@ -0,0 +1,342 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+
+    <div class="igOptions-horizontal">
+        <span class="igOptions-label">Type of Category Series: </span>
+                    <select value=@SeriesType
+                        @onchange="OnSeriesTypeChanged">
+                        <option>Column</option>
+                        <option>Area</option>
+                        <option>Bar</option>
+                        <option>Point</option>
+                        <option>Line</option>
+                        <option>Spline</option>
+                        <option>SplineArea</option>
+                        <option>StepArea</option>
+                        <option>StepLine</option>
+                        <option>Waterfall</option>
+                    </select>
+
+        <span class="igLegend-title">Legend: </span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
+    </div>
+
+    <div class="igComponent">
+        @if (DataSource != null)
+        {
+        <DataChart Height="100%" Width="100%"
+                   ChartTitle="Total Olympic Medals"
+                   Subtitle="United States vs China vs Russia"
+                   TitleTopMargin="10"
+                   IsHorizontalZoomEnabled="true"
+                   IsVerticalZoomEnabled="true"
+                   Legend="@LegendRef"
+                   @ref="DataChartRef">
```

Comment:
there should be 10 @if (ChartType == "Column" ) conditions each with one data chart for 10 types of series:
Column, Area, Bar, Point, Line, Spline, SplineArea, StepArea, StepLine, Waterfall

NOTE that I said 10 data charts because you will not be able to share axes between Bar series and rest of types of category series, e.g.
```
@if (ChartType == "Bar" )
{  
     <DataChart Height="100%" Width="100%" >
           <NumericXAxis />    
           <CategoryYAxis  />
           <BarSeries  /> 
      </DataChart>
}
@if (ChartType == "Area" || ChartType == "Column" || ChartType == "Point" || ChartType == "Line" || ChartType == "Spline" || ChartType == "SplineArea" || ChartType == "StepArea" || ChartType == "StepLine" || ChartType == "Waterfall")
{  
     <DataChart Height="100%" Width="100%" >
           <CategoryXAxis  />
           <NumericYAxis />    
           @if (ChartType == "Area" )
           {    
		<ColumnSeries  /> 
           }  
           @if (ChartType == "Column" )
           {    
		<ColumnSeries  /> 
           }
           @if (ChartType == "Point" )
           {    
		<PointSeries  /> 
           }
           @if (ChartType == "Line" )
           {    
		<LineSeries  /> 
           }
           @if (ChartType == "Spline" )
           {    
		<SplineSeries  /> 
           }
           @if (ChartType == "SplineArea" )
           {    
		<SplineAreaSeries  /> 
           }
           @if (ChartType == "StepArea" )
           {    
		<StepAreaSeries  /> 
           }
           @if (ChartType == "StepLine" )
           {    
		<StepLineSeries  /> 
           }
           @if (ChartType == "Waterfall" )
           {    
		<WaterfallSeries  /> 
           }
      </DataChart>
}```

Reply:
pending